packages/manager/apps/dedicated/client/app/dedicated/server/details/server.controller.js

```diff
@@ -352,6 +354,8 @@ export default class ServerCtrl {
       this.launchBringYourOwnImagePolling();
     }
 
+    this.firewallAsaInformations();
```

Comment:
**question:** Shouldn't be part of the `$q.all`?

Reply:
It's applied.